diff --git a/customer_app/bl602_sharedApp/proj_config.mk b/customer_app/bl602_sharedApp/proj_config.mk
index 964b891e..21a526a0 100644
--- a/customer_app/bl602_sharedApp/proj_config.mk
+++ b/customer_app/bl602_sharedApp/proj_config.mk
@@ -28,6 +28,7 @@ CONFIG_CLI_CMD_ENABLE:=0
 CONFIG_ENABLE_VFS_ROMFS:=1
 CONFIG_ADC_ENABLE_TSEN:=1
 CONFIG_EASYFLASH_ENABLE:=1
+CONFIG_BL602_USE_1M_FLASH := 1
 CONFIG_BL602_USE_ROM_DRIVER:=1
 CONFIG_FREERTOS_TICKLESS_MODE:=0
 
