diff --git a/src/driver/drv_ir2.c b/src/driver/drv_ir2.c
--- a/src/driver/drv_ir2.c
+++ b/src/driver/drv_ir2.c
@@ -329,10 +329,10 @@ static commandResult_t CMD_IR2_SetupIR2(const void* context, const char* cmd, co
 	else {
 		ADDLOG_ERROR(LOG_FEATURE_IR, (char *)"bk_timer driver not initialised?");
 		if ((int)res == -5) {
 			ADDLOG_INFO(LOG_FEATURE_IR, (char *)"bk_timer sddev not found - not initialised?");
-			return;
+			return CMD_RES_ERROR;
 		}
-		return;
+		return CMD_RES_ERROR;
 	}
 
 
